data/json/items/book/traps.json

```diff
@@ -52,9 +52,8 @@
   {
     "id": "textbook_traps",
     "type": "BOOK",
-    "name": "The Compleat Trapper",
-    "name_plural": "The Compleat Trapper",
-    "//": "The spelling is intentionally archaic, as this form is commonly used in the titles of books.",
```

Comment:
Has nobody ever heard of [The Compleat Angler](https://en.wikipedia.org/wiki/The_Compleat_Angler)?